src/data-catalog/DataBrowser.test.ts

```diff
@@ -1,7 +1,8 @@
 import * as _ from 'lodash/fp';
-import { convertColsToSettings, convertSettingsToCols, extractCatalogFilters } from 'src/pages/library/DataBrowser';
-import { datasetAccessTypes } from 'src/pages/library/dataBrowser-utils';
-import { TEST_DATASET_ONE, TEST_DATASET_TWO, TEST_DATASETS } from 'src/pages/library/test-datasets';
+
+import { datasetAccessTypes } from './data-browser-utils';
```

Comment:
It looks like we use both `./foo` and `foo` for relative paths. Is there an accepted convention for this in terra-ui? AFAIK the origin of the convention of `./` is due to the convention of removing `.` from `PATH` years ago, so I don't see what benefit it provides in other context like this.

Reply:
We used to use absolute imports (`src/...`) for everything. That doesn't work for code inside shared packages, so we've started to shift away from it.

The way I've been thinking about it is that eventually we want to have all of these higher level "feature areas" of the app become packages themselves. And thus, eventually we'll get to relative imports within a package and absolute imports (using the package name instead of `src`) for imports between packages.